contrib/win32/openssh/OpenSSHTestHelper.psm1

```diff
@@ -162,8 +162,14 @@ WARNING: Following changes will be made to OpenSSH configuration
        New-Item -ItemType Directory -Path $TestDataPath -Force -ErrorAction SilentlyContinue | out-null
     }
 
-    #Backup existing OpenSSH configuration
+    
+    if(-not (Test-Path $OpenSSHConfigPath -pathType Container))
+    {
+        #starting the service will create ssh config folder
+        start-service sshd
```

Comment:
These folders will also be created during install-sshd.ps1.

so lets throw an error "user should run install-sshd.ps1" and stop script when this folder doesn't exist.

Reply:
this is for test run on windows inbox where install-sshd.ps1 is not run